sale_channel_search_engine/models/se_binding.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2023 Akretion (https://www.akretion.com).
+# @author Raphaël Reverdy<raphael.reverdy@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class SeBinding(models.Model):
+    _inherit = "se.binding"
+
+    channel_id = fields.Many2one("sale.channel", compute="_compute_sale_channel")
+
+    def _compute_sale_channel(self):
+        for rec in self:
+            rec.channel_id = rec.backend_id.sale_channel_id[:1]
```

Comment:
the [:1] is not needed as sale_channel_id can only return an maximum one record (due to the constraint)